docs/source/best-practices/modularize-a-schema.md

```diff
@@ -0,0 +1,320 @@
+---
+title: Managing Large Schema
+description: Scaling your Apollo Server from a single file to your entire team
```

Comment:
We should modify this to match: https://dev-blog.apollodata.com/modularizing-your-graphql-schema-code-d7f71d5ed5f2

1. Let's not deal with circular deps, and just import everything into one central file
2. Talk about how to use `extend type` to extend the `Query` and `Mutation` types